components/server/src/workspace/workspace-starter.ts

```diff
@@ -859,6 +859,21 @@ export class WorkspaceStarter {
                 }
             }
 
+            const wsConnectionLimitingEnabled = await getExperimentsClientForBackend().getValueAsync(
+                "workspace_connection_limiting",
+                false,
+                { user },
```

Comment:
Sorry for the additional turn-around, but: I think It make sense to also sent `teams`. We already fetch it below (line 878) so we can just pull it up. :+1: